migrations/3_deploy_stakecontainer.js

```diff
@@ -21,7 +21,9 @@ module.exports = (deployer, network) => {
         throw new Error('No erc20TokenAddress & initialFees defined for this network')
     }
 
-    await deployer.deploy(ERC900BasicStakeContainer, codexCoinAddress)
+    // 3 months (in seconds), based on a 30 day month
```

Comment:
This is a next level nit but I'd write `90 days (in seconds)` since it's not 3 consecutive months :)